src/main/java/com/amihaiemil/docker/RtDocker.java

```diff
@@ -48,7 +50,14 @@
      * @param req HTTP Request. (see {@link Request})
      */
     RtDocker(final Request req) {
-        this.entry = req;
+        this.entry = req.header("Connection", "close");
```

Comment:
Connection: close header, otherwise the connection will stay open (this is the HTTP default) and Docker will send back a 400 BAD REQUEST after the initial response.